ds-prep-python-loops/main.py

```diff
@@ -0,0 +1,41 @@
+record = ("1", "Grimdiana", "Bones", "boulders")
+row = " "
+for x in record:
+  row = (row + x + ",")
+  print(row)
```

Comment:
The value of `row` should only be printed once, at the end:

---

![image](https://user-images.githubusercontent.com/16143771/165217401-c74c6025-91d5-46b7-b638-35c03a550ac8.png)

Reply:
But doesn't it say in the step before that to concatenate row, the value and a comma as a string and reassign all of that back to row